packages/app-project/stores/User/UserPersonalization/Notifications/Notifications.js

```diff
@@ -89,7 +116,8 @@ const Notifications = types
 
       reset () {
         this.unsubscribeFromSugarNotifications()
-        self.count = undefined
+        self.unreadConversationsIds = []
+        self.unreadNotificationsCount = null
```

Comment:
Maybe default the unread count to 0?

```suggestion
        self.unreadNotificationsCount = 0
```